### ISSUE TITLE
create-breadboard requires firebase-auth

### ISSUE THREAD

**Issue by PaulKinlan** [breadboard-ai/breadboard#720]
I just did a `npm init @google-labs/breadboard` and I noticed that a default install requires firebase. It seems pretty heavyweight (is this something in the debugger?)

**Comment by dglazkov** [comment]
Yeah, that does seem pretty heavy-weight. I am pretty sure it's `credentials` node in the `node-nursery-web`.
